flask_fs/backends/gridfs.py

```diff
@@ -54,7 +54,14 @@ def read(self, filename):
         return f.read()
 
     def write(self, filename, content):
-        return self.fs.put(self.as_binary(content), filename=filename)
+        kwargs = {
+            'filename': filename
+        }
+
+        if hasattr(content, 'content_type'):
```

Comment:
You should test that it's not `None` because most of the time Werkzeug set it to `None` (never succed to have another value).
```python
if getattr(content, 'content_type'):
```

Reply:
Ok, done. On my end, I always get a proper "content_type" in "request.files" if the originating request is set properly (i.e. the "Content-Type" header is set, thus making Werkzeug guess it).